user/pages/03.Howtos/10.l3-high-availability/docs.en.md

```diff
@@ -20,13 +21,21 @@ taxonomy:
 
 * `keepalived` is an open source VRRP implementation
 
-## Considerations
+## Legacy vs New regions
+
+Our legacy regions (cbk, dbl) have a different SDN backed than our new regions (fes, dus2).
```

Comment:
Legacy vs. new sounds negative / biased. Should we just write about differences neutrally?